src/Exception/NotFoundException.php

```diff
@@ -8,6 +8,6 @@
 /**
  * No entry was found in the container.
  */
-interface NotFoundException extends ContainerException
+interface NotFoundException
```

Comment:
I guess it is all going the way psr-6 etc moved. You can probably ignore this comment then :+1:  .